handlers/misttriggers/recording_end.go

```diff
@@ -180,8 +181,8 @@ type RecordingEndPayload struct {
 	ConnectionStartTimeUnix   int
 	ConnectionEndTimeUnix     int
 	StreamMediaDurationMillis int64
```

Comment:
Deliberately not changing this until we see a giant one come in. There's no reason this should exceed `math.MaxInt` though